.github/workflows/template-janitor.yml

```
@@ -65,9 +70,33 @@ jobs:
         run: |
           rm -rf ${{ env.TEMPLATES_PATH }}
 
-          # Can we get that from a variabl?
+          # Can we get that from a variable?
           # Remove this workflow as it has fulfilled its purpose
           rm -rf .github/workflows/template-janitor.yml
```

```suggestion
          rm .github/workflows/template-janitor.yml
```
I think this one should fail if the file is missing to avoid forgetting this if the janitor is beeing renamed